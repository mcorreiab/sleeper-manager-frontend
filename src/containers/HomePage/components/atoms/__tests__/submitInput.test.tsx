import renderer from 'react-test-renderer';
import SubmitInput from '../submitInput';


it('should render the input without props', () => {
    const tree = renderer.create(<SubmitInput />).toJSON();
    expect(tree).toMatchSnapshot();
});