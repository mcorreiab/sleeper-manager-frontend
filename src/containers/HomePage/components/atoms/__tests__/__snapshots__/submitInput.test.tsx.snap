// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the input without props 1`] = `
<input
  className="bg-sm-orange rounded-lg w-full h-auto py-3 px-4 
    text-base border-0 text-white font-bold outline-none"
  type="submit"
  value="GO"
/>
`;
